.github/workflows/deploy_hold.yml

```diff
@@ -50,19 +50,19 @@ jobs:
     runs-on: ubuntu-latest
     environment: production # sets in github repo with reviewer requirement protection rule
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Download build-output-EU artifact
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
           name: build-output-EU
           path: dist/
       - name: Authenticate to Google Cloud
         id: auth
-        uses: google-github-actions/auth@v0
+        uses: google-github-actions/auth@v2
         with:
           credentials_json: '${{ secrets.GCP_SA_KEY }}'
       - name: Deploy to GCP Bucket
-        uses: google-github-actions/upload-cloud-storage@v0
+        uses: google-github-actions/upload-cloud-storage@v2
```

Comment:
have you tested anywhere that this action and the auth action above complete successfully? I know it's not possible from the forked repo since we don't have the creds there. if you need help with testing it, let me know!